packages/apps/frontera/src/routes/organizations/src/components/Columns/organizations/Cells/relationship/OrganizationRelationship.tsx

```diff
@@ -77,6 +77,9 @@ export const OrganizationRelationshipCell = observer(
               .map((option) => (
                 <MenuItem
                   key={option.value.toString()}
+                  data-test={`relationship-${option.label
+                    .toLowerCase()
+                    .replace(/\s+/g, '-')}`}
```

Comment:
can you replace the option.label with option.valeu and completely remove the string processing bit after it?